strings.txt

```diff
@@ -11586,6 +11620,25 @@ COMPOSER
 	SV	Kompositör
 	ZH_CN	作曲
 
+COMPOSERALBUMS
+	CS	Skladatel Alba
+	DA	Komponist Album
+	DE	Komponist Alben
```

Comment:
Please make this "Komponistenalben" in German (as used in `SETUP_RELEASE_TYPES_GROUP_OPTIONS_DESC`)